<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSwap2</class>
 <widget class="QWidget" name="NewSwap2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>46</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>580</width>
         <height>540</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>580</width>
         <height>540</height>
        </size>
       </property>
       <widget class="control::MwcLabelNormal" name="secFeeUnitsLabel">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>155</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>satoshi per byte</string>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="secTransFeeEdit">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>140</y>
          <width>186</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200000</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Secondary currency transaction fee</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>512</number>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="secRedeemLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>BTC redeem time</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="secTransFeeLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>155</y>
          <width>146</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>BTC transaction fee</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Offer expiration time</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="autoStart_2">
        <property name="geometry">
         <rect>
          <x>305</x>
          <y>210</y>
          <width>271</width>
          <height>156</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Funds lock time</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>23</x>
           <y>50</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>MWC</string>
         </property>
        </widget>
        <widget class="QLabel" name="secLockCurrencyLabel">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>104</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>BTC</string>
         </property>
        </widget>
        <widget class="QLabel" name="mwcLockTimeLabel">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>3 hours and 32 minutes</string>
         </property>
        </widget>
        <widget class="QLabel" name="secLockTimeLabel">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>104</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>5 hours and 32 minutes</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="autoStart">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>276</width>
          <height>156</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Number of confirmations</string>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>23</x>
           <y>50</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>MWC</string>
         </property>
        </widget>
        <widget class="control::MwcLineEditNormal" name="mwcBlocksEdit">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>35</y>
           <width>91</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200000</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Number of confirmations for MWC coin</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>512</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>185</x>
           <y>50</y>
           <width>46</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>blocks</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>185</x>
           <y>104</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>blocks</string>
         </property>
        </widget>
        <widget class="control::MwcLineEditNormal" name="secBlocksEdit">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>89</y>
           <width>91</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200000</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Number of confirmations for secondary currency</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>512</number>
         </property>
        </widget>
        <widget class="QLabel" name="secBlocksLabel">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>104</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>BTC</string>
         </property>
        </widget>
        <widget class="QPushButton" name="confNumberHelpButton">
         <property name="geometry">
          <rect>
           <x>235</x>
           <y>45</y>
           <width>26</width>
           <height>22</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Click for help</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources_desktop.qrc">
           <normaloff>:/img/HelpBtn@2x.svg</normaloff>:/img/HelpBtn@2x.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="reviewButton">
        <property name="geometry">
         <rect>
          <x>355</x>
          <y>500</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Continue and review your trade parameters</string>
        </property>
        <property name="text">
         <string>Review</string>
        </property>
       </widget>
       <widget class="control::MwcLabelSmall" name="label_19">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>395</y>
          <width>526</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>ElectrumX private instance URL. Private instance provides extra security for the trade</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="backButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>500</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Return back to first step</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="electrumXEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>415</y>
          <width>526</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200000</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Private ElectrumX hosp and port to connect</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>512</number>
        </property>
        <property name="placeholderText">
         <string>Private ElectrumX server &lt;host&gt;:&lt;port&gt; (otional)</string>
        </property>
       </widget>
       <widget class="control::MwcComboBox" name="offerExpirationCombo">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>20</y>
          <width>366</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Message exchange and offer expiration time</string>
        </property>
       </widget>
       <widget class="control::MwcComboBox" name="secRedeemCombo">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>366</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Time reserved to redeem the coins</string>
        </property>
       </widget>
       <widget class="QPushButton" name="secRedeemHelpButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>90</y>
          <width>26</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Click for help</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources_desktop.qrc">
          <normaloff>:/img/HelpBtn@2x.svg</normaloff>:/img/HelpBtn@2x.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="offerEpirationHelpButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>30</y>
          <width>26</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Click for help</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources_desktop.qrc">
          <normaloff>:/img/HelpBtn@2x.svg</normaloff>:/img/HelpBtn@2x.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="secTransFeeHelpButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>150</y>
          <width>26</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Click for help</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources_desktop.qrc">
          <normaloff>:/img/HelpBtn@2x.svg</normaloff>:/img/HelpBtn@2x.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="electrumXHelpButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>425</y>
          <width>26</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Click for help</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources_desktop.qrc">
          <normaloff>:/img/HelpBtn@2x.svg</normaloff>:/img/HelpBtn@2x.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>245</x>
          <y>475</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="control::MwcLabelProgress" name="progress_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>46</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelProgress</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabelProgress.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcComboBox</class>
   <extends>QComboBox</extends>
   <header>control_desktop/MwcComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>offerExpirationCombo</tabstop>
  <tabstop>secRedeemCombo</tabstop>
  <tabstop>secTransFeeEdit</tabstop>
  <tabstop>mwcBlocksEdit</tabstop>
  <tabstop>secBlocksEdit</tabstop>
  <tabstop>electrumXEdit</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>reviewButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources_desktop.qrc"/>
 </resources>
 <connections/>
</ui>
